Forge.TreeWalker/src/TreeSchemaValidator.cs

```diff
@@ -0,0 +1,275 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="TreeSchemaValidator.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// <summary>
+//     The TreeSchemaValidator class implements the ITreeSchemaValidator interface.
+// </summary>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Json;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Forge.DataContracts;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
+
+
+namespace Microsoft.Forge.TreeWalker
+{
+    public class TreeSchemaValidator : ITreeSchemaValidator
```

Comment:
>public class TreeSchemaValidator [](http://example.com/codeflow?start=4&length=32)

Let's make this a static class. We want users to have the smoothest experience possible, and calling a static method is easy. We shouldn't need to instantiate an object since we aren't storing anything, and we get all the needed input from the user.